.github/workflows/functional_test_v2.yaml

```diff
@@ -60,6 +60,8 @@ jobs:
           cd functional_tests
           go test -v
   eks-test:
+    needs: kubernetes-test
+    if: github.repository == 'signalfx/splunk-otel-collector-chart'
```

Comment:
Does this change also need to be added below line 107?

Reply:
yes good catch, adding